runtime/reconcilers/model.go

```diff
@@ -276,10 +273,27 @@ func (r *ModelReconciler) Reconcile(ctx context.Context, n *runtimev1.ResourceNa
 		return runtime.ReconcileResult{Err: ctx.Err()}
 	}
 
-	// Build the output
-	execRes, execErr := executor.Execute(ctx)
+	var (
+		executorConnector string
+		execRes           *drivers.ModelResult
+		execErr           error
+	)
+	// Open executor for the new output and build the output
```

Comment:
nit: prefer comments that precede a full code section

```suggestion
	// Open executor for the new output and build the output
	var (
		executorConnector string
		execRes           *drivers.ModelResult
		execErr           error
	)
```